Packs/CommonScripts/Scripts/UnzipFile/UnzipFile.yml

```diff
@@ -36,15 +36,15 @@ outputs:
   description: A list of file names that were extracted from the ZIP file.
   type: Unknown
 script: '-'
+subtype: python3
 system: false
 tags:
 - Utility
 - file
 timeout: '0'
 type: python
-dockerimage: demisto/7zip
+dockerimage: devdemisto/unzip:1.0.0.10195
```

Comment:
a reminder to change after merging this: https://github.com/demisto/dockerfiles/pull/2295